docs/source/cleanlab/datalab/guide/table.rst

```diff
@@ -0,0 +1,186 @@
+.. tabs::
+
+    .. tab:: Classification task
+
+        .. list-table::
+            :widths: 20 10 20 50
+            :header-rows: 1
+
+            * - Issue Name
+              - Default
+              - Column Name
+              - Required keyword arguments in :py:meth:`Datalab.find_issues <cleanlab.datalab.datalab.Datalab.find_issues>`
+            * - :ref:`label<Label Issue>`
+              - YES
+              - 
+                :ref:`is_label_issue<\`\`is_label_issue\`\`>`
+
+                :ref:`label_score<\`\`label_score\`\`>` 
+
+                :ref:`given_label<\`\`given_label\`\`>`
+
+                :ref:`predicted_label<\`\`predicted_label\`\`>`
+              - `pred_probs` or `features`
+            * - :ref:`outlier<Outlier Issues>`
+              - YES
+              - 
+                :ref:`is_outlier_issue<\`\`is_outlier_issue\`\`>`
+                
+                :ref:`outlier_score<\`\`outlier_score\`\`>`
+              - `pred_probs` or `features` or `knn_graph`
+            * - :ref:`near_duplicate<(Near) Duplicate Issue>`
+              - YES
+              - 
+                :ref:`is_near_duplicate_issue<\`\`is_near_duplicate_issue\`\`>`
+                
+                :ref:`near_duplicate_score<\`\`near_duplicate_score\`\`>`
+
+                :ref:`near_duplicate_sets<\`\`near_duplicate_sets\`\`>`
+
+                :ref:`distance_to_nearest_neighbor<\`\`distance_to_nearest_neighbor\`\`>`
+              - `features` or `knn_graph`
+            * - :ref:`non_iid<Non-IID Issue>`
+              - YES
+              - 
+                :ref:`is_non_iid_issue<\`\`is_non_iid_issue\`\`>`
+                
+                :ref:`non_iid_score<\`\`non_iid_score\`\`>`
+              - `pred_probs` or `features` or `knn_graph`
+            * - :ref:`class_imbalance<Class Imbalance Issue>`
+              - YES
+              - 
+                :ref:`is_class_imbalance_issue<\`\`is_class_imbalance_issue\`\`>`
+                
+                :ref:`class_imbalance_score<\`\`class_imbalance_score\`\`>`
+              - None [#f1]_
+            * - :ref:`underperforming_group<Underperforming Group Issue>`
+              - YES
+              - 
+                :ref:`is_underperforming_group_issue<\`\`is_underperforming_group_issue\`\`>`
+                
+                :ref:`underperforming_group_score<\`\`underperforming_group_score\`\`>`
+              - (`pred_probs`, `features`) or (`pred_probs`, `knn_graph`) or (`pred_probs`, `cluster_ids`) [#f2]_
+            * - :ref:`null<Null Issue>`
+              - YES
+              - 
+                :ref:`is_null_issue<\`\`is_null_issue\`\`>`
+                
+                :ref:`null_score<\`\`null_score\`\`>`
+              - `features`
+            * - :ref:`data_valuation<Data Valuation Issue>`
+              - no
```

Comment:
```suggestion
              - NO
```

Reply:
I'd actually suggest make the yes's: `Yes` and this one `No` vs. all caps.
But definitely not some all-caps some not.

Reply:
Done.